toolchain/sem_ir/inst_fingerprinter.cpp

```diff
@@ -359,6 +376,7 @@ struct Worklist {
         Add(inst.type_id());
       }
 
+      // TODO: Exclude the decl block of a declaration.
```

Comment:
Don't the changes in this PR already do this?

Reply:
Yes, oops, forgot to delete this TODO :) Thanks.
```suggestion
```